src/config.ts

```diff
@@ -2,3 +2,5 @@ export const ACCESS_TOKEN_KEY = 'eh_token_2020_11_22'
 export const EGGHEAD_USER_COOKIE_KEY = 'eh_user'
 export const CIO_IDENTIFIER_KEY = 'cio_id'
 export const CIO_CUSTOMER_OBJECT_KEY = 'cio_customer'
+
+export const ARTICLE_NEWSLETTER_INTEREST_KEY = 'article_cta_signed_up'
```

Comment:
Continuing the pattern used above for creating the cookie name

Reply:
Having a dedicated cookie made the most sense since some existing users may be interested in the email courses offered. So I didn't want to hide the CTA based on subscriber status